recipe_scrapers/streetkitchen.py

```diff
@@ -21,11 +30,10 @@ def image(self):
         )
 
     def ingredients(self):
-        ingredients_raw = self.soup.findAll("dd")
+        ingredients_raw = self.soup.find("div", class_="ingredients-main").findAll("dd")
         ingredients = []
-        # There are separate sets of ingredients for desktop and mobile view
-        for ingredient in ingredients_raw[: int(len(ingredients_raw) / 2)]:
-            ingredients.append(normalize_string(ingredient.get_text()).strip())
+        for ingredient in ingredients_raw:
+            ingredients.append(normalize_string(ingredient.get_text()))
```

Comment:
Optional suggestion: I find `.text` slightly more convenient and readable (shorter, and fewer parentheses)

```suggestion
            ingredients.append(normalize_string(ingredient.text))
```